velox/expression/CastExpr.cpp

```diff
@@ -688,6 +688,18 @@ void CastExpr::evalSpecialForm(
   inputs_[0]->eval(rows, context, input);
   auto fromType = inputs_[0]->type();
   auto toType = std::const_pointer_cast<const Type>(type_);
+  // In a TryExpr context, all the inputs could be errors. We skip evaluation
+  // and return a NULL result vector in this case.
+  const auto errorVec = context.errors();
+  if (errorVec &&
+      BaseVector::countNulls(errorVec->nulls(), errorVec->nulls()->size()) ==
```

Comment:
Perhaps, use EvalCtx::deselectErrors to skip evaluation if all requested rows have errors.